Makefile

```diff
@@ -17,9 +17,11 @@ TRACE ?= off
 # Specify pool malloc or heap malloc
 MALLOC ?= pool
 
+# TODO: The latest Zephyr seems to require running make mrproper for each build
+#       the mrproper target should be removed when this is fixed
 # Build for zephyr, default target
 .PHONY: zephyr
-zephyr: analyze generate
+zephyr: mrproper analyze generate
```

Comment:
It turns out that running "make mrproper" is not very detrimental to the build time (unlike e.g. make clean), so we may just want to leave this here.